todo_app/app.py

```diff
@@ -25,9 +48,20 @@ def index():
             items.sort(key=lambda x: x.due_date)
         elif request.values.get('sort') == '2':
             items.sort(key=lambda x: x.due_date, reverse=True)
-        return render_template("index.html",View_Model=get_view_model, todays_date=todays_date)
+
+        if hasattr('current_user', 'role'):
```

Comment:
Currently they is checking _the string_ "current_user" so it always comes up false!
```suggestion
        if hasattr(current_user, 'role'):
```